Gifski/EditVideoViewController.swift

```diff
@@ -310,6 +320,20 @@ final class EditVideoViewController: NSViewController {
 	}
 
 	private func estimateFileSize() {
+		// Cancle old estimate
+		estimateWorkItem?.cancel()
```

Comment:
maybe we could extract these 4 lines to a method e.g. `cancelEstimationProcess()` and then we could remove this comment above as well?